source/configuration_manual/mail_location/obox/http_storages.rst

```diff
@@ -82,7 +82,13 @@ The parameters common to all object storages include:
 +---------------------------------------+-------------------------------------------------------------------------------------------------------------------------------+--------------+
 | max_connect_retries=<n>               |Number of connect retries                                                                                                      | 2            |
 +---------------------------------------+-------------------------------------------------------------------------------------------------------------------------------+--------------+
-| max_retries=<n>                       |Max number of HTTP request retries                                                                                             | 4            |
+| max_retries=<n>                       |Max number of HTTP request retries. Retries happen for 5xx errors as well as for 423(locked)                                   | 4            |
+|                                       |with :ref:`sproxyd <scality_sproxyd>` and 409(conflict) with :ref:`cdmi <scality_cdmi>`.                                       |              |
+|                                       |There is a wait between attempting next retry. The initial retry wait is 50ms, followed by doubling 100ms, 200ms, 400ms and    |              |
+|                                       |so forth.                                                                                                                      |              |
+|                                       |                                                                                                                               |              |
+|                                       |.. versionchanged:: 2.3.15 The new retry times are increased by multiplying with 10 e.g: 50ms -> 500ms -> 5s -> 10s. The       |              |
```

Comment:
Would it make more sense to describe the new behavior in the main paragraph and the old behavior in this versionchanged line?